File: Backend/OnlineNetBankingWebAPI/Models/Payee.cs
using System;
using System.Collections.Generic;

namespace OnlineNetBankingWebAPI.Models;

public partial class Payee
{
    public int PayeeId { get; set; }

    public int? SenderId { get; set; }

    public string? PayeeEmail { get; set; }

    public string? SenderEmail { get; set; }

    public string? PayeeAccNo { get; set; }

    public string? SenderAccNo { get; set; }

    public virtual UserDetail? Sender { get; set; }
}

File: user-portal/src/app/Models/payee.ts
export class Payee {
    payeeId : number = 0;
    senderId : number = 0;
    payee_email? : string = "";
    sender_email : string = "";
    payee_acc_no : string = "";
    sender_acc_no : string = "";


    constructor(payeeId: number, senderId: number, payeeEmail: string, senderEmail: string, payeeAccNo:string,senderAccNo:string){
     this.payeeId = payeeId;
     this.senderId = senderId;
     this.payee_email = payeeEmail;
     this.sender_email= senderEmail;
     this.payee_acc_no = payeeAccNo;
     this.sender_acc_no = senderAccNo;
    }
}
